content/source/docs/enterprise/install/uninstall.html.md

```diff
@@ -21,9 +21,28 @@ After you initiate `uninstall`:
 
 	```
 	$ sudo ./uninstall.sh
-	This script will completely uninstall Terraform Enterprise and Replicated on this system, as well as remove associated files. This includes the Replicated snapshot directory. Please back up /var/lib/replicated/snapshots if you wish to save them.
+	This script will completely uninstall Terraform Enterprise and Replicated on this system, as well as remove associated files.
 	Do you wish to continue? (y/n)yes
 	Proceeding with uninstall...
+	```
+
+	If there are snapshots present on the system, you will be prompted to move them, proceed with deleting them, or abort.
+	
+	```
+	There appear to be Replicated snapshots stored in /var/lib/replicated/snapshots.
+	1) Move the snapshots to another directory
+	2) Continue uninstall and delete the snapshots
+	3) Cancel the uninstall
+	Select an option: 1
+	Enter the directory to move the snapshots to: /tmp
+	The snapshots will be moved to /tmp.
+	Press Y to continue or N to cancel. y
+	Moving snapshots...
+	Files moved.
+	```
+	The uninstall will then continue...
```

Comment:
I would do this here, and remove that line from below the example block. I think this is clearer. 

```suggestion
	The script continues and stops the Replicated services, removes the Docker containers, and removes Replicated executables and configuration files from the system.  
```